setup.json

```
@@ -92,7 +92,7 @@
     },
     "install_requires": [
         "aiida_core[atomic_tools]~=1.4,>=1.4.4",
-        "aiida-pseudo~=0.6.0",
+        "aiida-pseudo~=0.6,>=0.6.1",
```

these are not really the same, now it will also allow `aiida-pseudo==0.7` and higher. whereas before it would only install the patches of 0.6 series. Instead you could do, 
```suggestion
        "aiida-pseudo~=0.6.1",
```
which should achieve what you intended I think

---

Hmm, I see, thanks! Will this also upgrade `aiida-pseudo` from `0.6.0` to `0.6.1` when installing the new `aiida-quantumespresso` release?

---

yes, I `~=0.6.1` literally is identical to `>= 0.6.1, == 0.6.*`. See [PEP 440](https://www.python.org/dev/peps/pep-0440/#compatible-release)